### PR TITLE
GalleryViewController + HelpViewController

### PR DESCRIPTION
**Pull request by kobsonaut**
Adding GalleryViewController + HelpViewController

### CODE REVIEW

File: TOZ_iOS/GalleryTableViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+
+        // Configure the view for the selected state
+    }
```
Comment (ArkadiuszBanas) [inline TOZ_iOS/GalleryTableViewCell.swift]: We don't need override this method if we are not using them.

File: TOZ_iOS/GalleryViewController.swift [hunk 2]
```diff
@@ -7,10 +7,48 @@
...
-
+    
+    // UITableViewDataSource
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return animalsArray.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        
+        let galleryCell: GalleryTableViewCell = tableView.dequeueReusableCell(withIdentifier: "galleryTableViewCell") as! GalleryTableViewCell
```
Comment (ArkadiuszBanas) [inline TOZ_iOS/GalleryViewController.swift]: Forcing casting is forbidden.

File: TOZ_iOS/GalleryViewController.swift [hunk 3]
```diff
@@ -7,10 +7,48 @@
...
+        
+        let animalInfo = animalsArray[indexPath.row]
+        
+        galleryCell.animalDescription.text = animalInfo.description
```
Comment (ArkadiuszBanas) [inline TOZ_iOS/GalleryViewController.swift]: I suggest to pass animalInfo to cell object and customize subview inside cell. Please take a look on news view controller.

File: TOZ_iOS/GalleryViewController.swift [hunk 4]
```diff
@@ -7,10 +7,48 @@
...
+        galleryCell.animalName.text = animalInfo.name
+        galleryCell.animalAge.text = animalInfo.age
+        galleryCell.animalImage.image = animalInfo.image
+        return galleryCell
+    }
+    
+    // UITableViewDelegate
+    func tableView(_ tableView: UITableView, didDeselectRowAt indexPath: IndexPath) {
+        
+    }
```
Comment (ArkadiuszBanas) [inline TOZ_iOS/GalleryViewController.swift]: Remove it.

File: TOZ_iOS/HelpFinanceViewController.swift [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Do any additional setup after loading the view.
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
```
Comment (ArkadiuszBanas) [inline TOZ_iOS/HelpFinanceViewController.swift]: We don't need override this method if we are not using them.

File: TOZ_iOS/GalleryViewController.swift [hunk 6]
```diff
@@ -7,10 +7,48 @@
 
 import UIKit
 
-class GalleryViewController: UIViewController {
-
+class GalleryViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
```
Comment (wobrak) [inline TOZ_iOS/GalleryViewController.swift]: Maybe it would look better by using extension for UITableVIewDataSource, UITabeVIewDelegate

**Pull request merged**